meinberlin/apps/contrib/tasks.py

```diff
@@ -28,7 +28,7 @@ def download_footer(footer_url, filepath) -> bool:
         return True
 
 
-@shared_task
+@shared_task(name="periodic_footer_update")
```

Comment:
thanks, sorry my example was wrong :/

Reply:
uhm no, this is needed for shared tasks, to give the name. I forgot it.